CHANGELOG.md

```diff
@@ -66,6 +66,8 @@ Main (unreleased)
 
 - Flow: `prometheus.relabel` and `prometheus.remote_write` will now error if they have exited. (@ptodev)
 
+- Agent Management: Fix `remote_config_fetch_errors_total` metric being registered even when `remote-configs` and `agent-management` features were not enabled. (@spartan0x117)
```

Comment:
Since this PR is changing code that hasn't been released yet, it wouldn't go into the changelog (e.g., the changelog should only include things which have changed since the last release)